README.rst

```diff
@@ -3,28 +3,33 @@ Introduction
 
 docky : Make docker and docker-compose simply !
```

Comment:
this 1st sentence is non English, this is bad to frighten international users right at the start.
I propose to simply remove it, as the next sentence is a good introduction.